sdk/spring/azure-spring-integration-test/src/main/java/com/azure/spring/integration/test/support/SendOperationTest.java

```diff
@@ -78,6 +80,26 @@ public void testSendWithPartitionKey() throws ExecutionException, InterruptedExc
         verifyGetClientCreator(1);
     }
 
+    @Test
```

Comment:
We can ut for the message converter to check if the header of paritionkey is set as expected.